src/assets/js/init.js

```diff
@@ -88,7 +88,7 @@ window.onload = function() {
     var tmp = w.navigator.languages && w.navigator.languages[0]
       || w.navigator.language || w.navigator.userLanguage;
     tmp = tmp.split('-')[0];
-    for (var i=0, l=langs; i < l; i++) {
+    for (var i=0, l=langs.length; i < l; i++) {
```

Comment:
It doesn't make much sense to create a separate variable `l` here since it isn't being used inside the for loop. Can it instead be like below:
```suggestion
    for (var i=0; i < langs.length; i++) {
```

Reply:
Yes, It can be done.
In the complete init.js file, all the functions are using one extra variable `l` to store value
So this can be changed as well as all the extra variables used in other functions also

Reply:
If it is the same case where `l` is not used elsewhere then do replace those patterns with the simpler one like the above. Thanks.

Reply:
Kindly have a look at latest commit!